apimatic_core/utilities/api_helper.py

```diff
@@ -668,6 +676,27 @@ def to_lower_case(list_of_string):
 
         return list(map(lambda x: x.lower(), list_of_string))
 
+    @staticmethod
+    def get_additional_properties(dictionary, unboxing_function):
+        """Extracts additional properties from the dictionary.
+
+        Args:
+            dictionary (dict): The dictionary to extract additional properties from.
+            unboxing_function (callable): The deserializer to apply to each item in the dictionary.
+
+        Returns:
+            dict: A dictionary containing the additional properties and their values.
+        """
+        additional_properties = {}
+        for key, value in dictionary.items():
```

Comment:
How will the unboxing function will behave incase of value as array or dictionary given we are passing the function **lambda x: str(x))**

Reply:
it is already known like what will be the type of the additional property values, therefore I handled it accordingly.